bvm/ballerina-observe/src/main/java/org/ballerinalang/observe/trace/SpanStore.java

```diff
@@ -42,6 +42,10 @@
         return spans.get(spanId);
     }
 
+    Map<String, SpanContext> getSpanContext(String spanId) {
```

Comment:
Wrong method name.

Reply:
Fixed in 4ffbfc6ad9a11cc24598f426352bc7c86b4b4724